root/static/scripts/release-editor/validation.js

```diff
@@ -37,7 +37,7 @@ function markTabWithErrors($panel) {
   var tabs = releaseEditor.uiTabs;
 
   var $errors = $('.field-error', $panel).filter(function () {
-    return $(this).data('visible') && $(this).text();
+    return String($(this).data('visible')) === 'true' && $(this).text();
```

Comment:
This changed because we now have, e.g., `data-visible={String(missingEditNote)}` on the elements themselves.

Previously we were setting `visible` via jQuery like `$element.data('visible', bool)`. But this does not affect the `data-` HTML attribute on the element. [1]

Despite the `.data()` *setter* not affecting the HTML attribute, confusingly, the getter *does read* from the HTML attribute! [1]

[1] https://api.jquery.com/data/